tools/profiling/microbenchmarks/bm_diff/requirements.txt

```diff
@@ -0,0 +1,4 @@
+tabulate>=0.8
+scipy>=1.0.0
+requests>=2.19.0
+jwt>=0.5.0
```

Comment:
No, the tools/gce/linux_kokoro_performance_worker_init.sh file installs dependencies for CI. This file is for human convenience, since I had to figure out what these requirements were myself when I wanted to run the benchmarks locally.

Reply:
Correction: I liked that idea, and changed the test setup script to utilize this requirements file. There's some definite redundancy in our python setup scripts though, I had to change the cryptography constraints in 3 different files for example. It's not trivial to deduplicate them, so I think that should be a cleanup for another time as well.